node_modules/oae-authentication/config/strategies.js

```diff
@@ -32,7 +32,7 @@ module.exports = {
             'enabled': new Fields.Bool('Google Authentication Enabled', 'Allow Google authentication for tenant', false),
             'key': new Fields.Text('Google client ID', 'Google client ID', '822036948319.apps.googleusercontent.com', {'suppress': true}),
             'secret': new Fields.Text('Google client secret', 'Google client secret', 'b5ccr5noy0j_gzj4kj42uuwm', {'suppress': true}),
-            'hostedDomain': new Fields.Text('Google hosted domain', 'Google hosted domain. e.g., Google apps domain (optional)', '')
+            'domains': new Fields.Text('Google domain(s)', 'A comma-separated lists of allowed email domains (optional)', '')
```

Comment:
Yes, I made a note for that in the wiki https://github.com/oaeproject/Hilary/wiki/OAE-Upgrade-Guide#google-authentication